model/src/main/java/com/basistech/rosette/dm/ConvertFromPreAdm1.java

```diff
@@ -0,0 +1,252 @@
+/*
+* Copyright 2016 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.dm;
+
+import com.basistech.rosette.RosetteRuntimeException;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utility class with methods for conversion of the data from
+ * the pre-1.0.0 data model to the 1.0.0 data model.
```

Comment:
You mean pre-1.1.0 to 1.1.0?  We should make it clear that anything without
a version is treated as 1.0.0.